src/main/java/com/mojang/brigadier/context/ParsedArgument.java

```diff
@@ -16,7 +16,7 @@
     private final T result;
 
     /**
-     * Creates a new ParsedArgument for a given string range with a  given value.
+     * Creates a new {@code ParsedArgument} for a given string range with a  given value.
```

Comment:
No `{@code ...}`. Otherwise it is inconsistent with other methods and constructors.